include/rmm/cuda_stream_view.hpp

```diff
@@ -33,19 +33,31 @@ namespace rmm {
  */
 class cuda_stream_view {
  public:
-  constexpr cuda_stream_view()                                   = default;
-  constexpr cuda_stream_view(cuda_stream_view const&)            = default;
-  constexpr cuda_stream_view(cuda_stream_view&&)                 = default;
+  constexpr cuda_stream_view()                        = default;
+  constexpr cuda_stream_view(cuda_stream_view const&) = default;  ///< Default copy constructor
+  constexpr cuda_stream_view(cuda_stream_view&&)      = default;  ///< Default move constructor
+  /**
+   * @brief Default copy assignment operator
+   *
+   * @return cuda_stream_view& Reference to the assigned object
+   */
   constexpr cuda_stream_view& operator=(cuda_stream_view const&) = default;
-  constexpr cuda_stream_view& operator=(cuda_stream_view&&)      = default;
-  ~cuda_stream_view()                                            = default;
+  /**
+   * @brief Default move assignment operator
+   *
+   * @return cuda_stream_view& Reference to the assigned object
+   */
+  constexpr cuda_stream_view& operator=(cuda_stream_view&&) = default;
+  ~cuda_stream_view()                                       = default;
 
   // Disable construction from literal 0
   constexpr cuda_stream_view(int)            = delete;  //< Prevent cast from 0
   constexpr cuda_stream_view(std::nullptr_t) = delete;  //< Prevent cast from nullptr
 
   /**
    * @brief Implicit conversion from cudaStream_t.
+   *
+   * @param stream A cuda_stream_view that wraps the stream.
```

Comment:
This isn't right. The parameter is the underlying stream for this stream view.

Reply:
Ah you're right. I accidentally documented this as an implicit conversion operator instead of a constructor (that allows implicit conversion) as well.